.github/workflows/ci.yml

```diff
@@ -178,6 +178,7 @@ jobs:
       env:
         COLOR: 'yes'
         AIOHTTP_NO_EXTENSIONS: ${{ matrix.no-extensions }}
+        INSTALL_ARG: '--user'
```

Comment:
All of pip CLI options have env var counterparts prefixed with `PIP_`. For example, here we could use
```suggestion
        PIP_USER: 1
```
which is the same as `--user`: https://pip.pypa.io/en/latest/topics/configuration/#environment-variables.

Reply:
I'll give that a try then. Not sure if all commands should be --user or not, so we'll see.